File: 392. Is Subsequence/main.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        n=len(s)
        m=len(t)

        first=0;
        second=0;

        while first < n and second < m:
            if s[first]==t[second]:
                first=first+1

            second=second+1

        

        return first==n
        
        

File: 392. Is Subsequence/main.cpp
#include <string>

class Solution {
public:
    bool isSubsequence(std::string s, std::string t) {


        int first=0;
        int second=0;

        int subSize=s.size();
        int strSize=t.size();

        
        while(first< subSize &&  second < strSize)
        {
            if(s[first]==t[second])
            {
                ++first;
            }
            
            second++;
        }

        return first==subSize;
        
    }
};